src/main/scala/cognite/spark/v1/DataModelInstancesRelation.scala

```diff
@@ -74,44 +72,43 @@ class DataModelInstanceRelation(config: RelationConfig, modelExternalId: String)
       dmi: DataModelInstanceQueryResponse,
       requiredPropsArray: Seq[String]): ProjectedDataModelInstance = {
     val dmiProperties = dmi.properties.getOrElse(Map())
-    ProjectedDataModelInstance(
+    val res = ProjectedDataModelInstance(
```

Comment:
I don't think you need the `res` thing here, perhaps it's leftovers from development?

Reply:
yis